src/Firebase.Notifications/Common.uno

```diff
@@ -0,0 +1,169 @@
+using Uno.Compiler.ExportTargetInterop;
+using Uno;
+using Uno.Graphics;
+using Uno.Platform;
+using Uno.Collections;
+using Fuse;
+using Fuse.Controls;
+using Uno.Threading;
+
+namespace Firebase.Notifications
+{
+    [Require("Entity","Firebase.Core.Init()")]
+    [ForeignInclude(Language.Java,
+            "android.util.Log",
+            "com.google.firebase.iid.FirebaseInstanceId")]
+    [extern(iOS) Require("Source.Include", "Firebase/Firebase.h")]
+    public static class NotificationService
+    {
+        extern(Android)
+        static NotificationService()
```

Comment:
Android and iOS aren't the only two platforms; removing the "Firebase Notifications are not yet available on this platform"-message here doesn't seem right. You should probably update the extern for that platform to be something like `extern(!Android && !iOS)`.